src/Umbraco.Core/Configuration/Models/DeliveryApiSettings.cs

```diff
@@ -48,4 +48,40 @@ public class DeliveryApiSettings
     /// <value><c>true</c> if the Delivery API should output rich text values as JSON; <c>false</c> they should be output as HTML (default).</value>
     [DefaultValue(StaticRichTextOutputAsJson)]
     public bool RichTextOutputAsJson { get; set; } = StaticRichTextOutputAsJson;
+
+    /// <summary>
+    ///     Gets or sets the settings for the Media APIs of the Delivery API.
+    /// </summary>
+    public MediaSettings Media { get; set; } = new ();
+
+    /// <summary>
+    ///     Typed configuration options for the Media APIs of the Delivery API.
+    /// </summary>
+    /// <remarks>
+    ///     The Delivery API settings (as configured in <see cref="DeliveryApiSettings"/>) supersede these settings in levels of restriction.
+    ///     I.e. the Media APIs cannot be enabled, if the Delivery API is disabled.
+    /// </remarks>
+    public class MediaSettings
```

Comment:
It ShouldJustWork(tm) because the `MediaSettings` instance is part of the (already existing) `DeliveryApiSettings` instance. If autocomplete does not work for you, you might need to delete all your `appsettings-schema.Umbraco.Cms.json` files and rebuild the solution to generate new ones?

<img width="740" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/7405322/2da9b40b-07bb-4b36-a12b-42dff2128dea">

<img width="394" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/7405322/ab063576-2448-472c-b9f9-6f65b1dcb1b0">

Reply:
Yep, that solved it 💪